Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -388,6 +388,7 @@ class BrowserViewController: UIViewController {
 
         view.addSubview(alertStackView)
         footer = UIView()
+        footer.layer.zPosition = CGFloat.leastNormalMagnitude
```

Comment:
This is footing the `footer` on the top. We want to hide toast behind the footer view which contains the TabToolBar. Setting the `zPosition` little higher helps to achieve this.

I was not able to figure out another way to achieve this.

Reply:
Oh ok, I wasn't sure why the zPosition wasn't set to `1` or such, I hadn't seen `leastNormalMagnitude` used before, but I see it will compare as >0 and less than any other positive integer.